cogs/help.py

```diff
@@ -7,6 +7,10 @@
 
 
 class JDBotHelp(commands.MinimalHelpCommand):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
```suggestion
# from typing import Any

    def __init__(self, **options: Any):
        super().__init__(**options)
```

Reply:
Should I type the whole thing or just this part.

Reply:
that part should be fine for now.

Reply:
Done I think